com.unity.render-pipelines.universal/ShaderLibrary/ShaderVariablesFunctions.hlsl

```diff
@@ -224,6 +224,16 @@ half3 MixFog(real3 fragColor, real fogFactor)
     return MixFogColor(fragColor, unity_FogColor.rgb, fogFactor);
 }
 
+// Linear depth buffer value between [0, 1] or [1, 0] to eye depth value between [near, far]
+real LinearDepthToEyeDepth(real rawDepth)
```

Comment:
My opinion is that we should add an override on Common.hlsl of LinearEyeDepth that handles orthographic. It's rather confusing that we have a similar function name in URP library and none of those functions specific if ortho is supported or not. 

Also it seems this one takes rawDepth unreversed depth while the others in Common.hlsl take depth.

Reply:
We can't add an override as the value "space" is not encoded into the float in anyway, except by convention. This rescales linear depth (or reverse linear depth) into near<far view/eye space linear depth.
Functions in common.hlsl convert and rescale non-linear depth into view/eye space linear depth.
This also uses unity global constants directly, while common.hlsl functions are more generic and take everything as parameters.
REVERSE_Z-if is required to support OpenGL(ES).